internal/olm/operator/registry/operator_installer.go

```diff
@@ -290,9 +290,44 @@ func (o OperatorInstaller) createSubscription(ctx context.Context, csName string
 	return sub, nil
 }
 
-func (o OperatorInstaller) getInstalledCSV(ctx context.Context) (*v1alpha1.ClusterServiceVersion, error) {
-	c, err := olmclient.NewClientForConfig(o.cfg.RESTConfig)
```

Comment:
Instead of setting up these options, you can just do
```go
func (o OperatorInstaller) getInstalledCSV(ctx context.Context) (*v1alpha1.ClusterServiceVersion, error) {
	c := olmclient.Client{KubeClient: o.cfg.Client}
```

Reply:
Done